internal/constants.go

```diff
@@ -15,6 +15,8 @@ package internal
 
 const (
 	BootTimeoutDefault             = 30000
+	BootTimeoutSecondsDefault      = 30
```

Comment:
BootTimeoutSecondsDefault = BootTimeoutDefault/1000

Reply:
The reason I didn't do that is because very soon BootTimeoutDefault will no longer be used and therefore can just be deleted....probably could be made a separate issue to do the cleanup as dependencies on the 2 or 3 remaining usages.

Reply:
Little confused by comment about needing to multiply by seconds, looks like it's already there.
`func NewStartUpTimer(retryIntervalInSeconds, maxWaitInSeconds int) Timer {
	return Timer{
		startTime: time.Now(),
		duration:  time.Millisecond * time.Duration(maxWaitInSeconds),
		interval:  time.Millisecond * time.Duration(retryIntervalInSeconds),
	}
}
`

Reply:
so Internally StartupTimer changes the duration and interval to milliseconds so that comparisons to milliseconds the rest of the way works.

Reply:
> The reason I didn't do that is because very soon BootTimeoutDefault will no longer be used and therefore can just be deleted....probably could be made a separate issue to do the cleanup as dependencies on the 2 or 3 remaining usages.

These two constants are the same constant expressed in different units of measure -- one is in seconds and the other in milliseconds.  Explicitly tie them together and leave it to a follow-on PR to set BootTimeoutSecondsDefault to 30 when BootTimeoutDefault has all other references removed from the code.

Reply:
> Little confused by comment about needing to multiply by seconds, looks like it's already there.

It isn't.  The code is multiplying by `time.Millisecond` and not by `time.Second` which is why the code actually does timeout after 30 seconds (and not the 30,000 seconds as you reported). Even though the variable name denotes unit-of-measure as seconds, we're passing in milliseconds (and multiplying that value by time.Millisecond to get the common duration in nanoseconds.

Run master core data as described above without any changes and verify that it times out after 30 seconds.  Then run it again with the changes in this PR to verify it times out immediately.